.github/workflows/ci-workflow.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: Build core packages
+        run: pnpm build --filter=@suspensive/react --filter=@suspensive/react-query
+
```

Comment:
ci-workflow.yml will be responsible for only core packages (@suspensive/react, @suspensive/react-query). So it saves time on github actions.

@suspensive/docs and @suspensive/visualization are removed from this workflow because vercel can check for errors at build time.